dev/build_release.py

```diff
@@ -63,7 +63,7 @@
 from spinnaker.run import run_quick
 
 
-SUBSYSTEM_LIST = ['clouddriver', 'orca', 'front50',
+SUBSYSTEM_LIST = ['clouddriver', 'orca', 'front50', 'halyard',
```

Comment:
Is this enough? https://github.com/spinnaker/spinnaker/blob/master/dev/refresh_source.py#L94